xml/s4s_tune.xml

```diff
@@ -392,6 +391,255 @@
   </sect2>
  </sect1>
 
+ <sect1 xml:id="sec-sapconf-5">
+  <title>Tuning Systems with &sapconf; 5</title>
+  <para>
+   The package <systemitem>sapconf</systemitem> is available for the &x86-64;
+   and &ppc64le; architecture.
+   It sets recommended parameters for the following types of &sap; applications:
+   &netweaver;, &hana; and &hana;-based applications.
+  </para>
+  <para>
+   Note that if you previously made changes to the system tuning, those
+   changes may be overwritten by &sapconf;.
+  </para>
+  <para>
+   &sapconf; ships s &systemd; service which applies the tuning and ensures that
+   related services are running.
+  </para>
+  <para>
+   To use &sapconf;, make sure
+   that the package <package>sapconf</package> is installed on your system.
+  </para>
+  <note>
+   <title><command>sapconf</command> Utility Is Not Included in &productname; &productnumber;</title>
+   <para>
+    The command-line utility <command>sapconf</command> that was included in
+    &productname; 11 and 12 is not available anymore in
+    &productname; &productnumber;.
```

Comment:
This is about a shipped command called sapconf! Don't mix it up with the package.
In the old days (SysV+SLES11+SLES12SP0/1) the script did the tuning. With tuned this became obsolete and was marked as deprecated (which led to the misunderstanding that sapconf itself was going away!). 
Now with removing tuned, but still having profiles in SLES 12, we resurrect the command to set the sapconf profile.
In SLES 15 there are no profiles and the commandline tool is not needed and not even exist.
It is complicated, I know.

Reply:
Ok, I see. That sound to me like some implementation detail. I usually try to avoid that in the documentation (because that is usually something for the release notes).

It seems to me, we can leave it as is for the time being. If you have further additions, let me know. :wink: